docs/docs/faq.md

```diff
@@ -0,0 +1,60 @@
+# Frequently asked questions
+
+### Why is the `NEBARI_KUBECONFIG` file in `/tmp`?
+
+Nebari regenerates this file on every run. Yes, it will be removed by the operating system during its cleanup process, but running the `qhub deploy` command again as Nebari will update/create a `NEBARI_KUBECONFIG` file for you.
```

Comment:
```suggestion
Nebari regenerates this file on every run. This means it will be removed by the operating system during its cleanup process, but running the `qhub deploy` command again as Nebari will update/create a `NEBARI_KUBECONFIG` file for you.
```
Improve wording to make it more inclusive.

> but running the `qhub deploy` command again as Nebari

I find this confusing - so should I have a `nebari` user to call this command? if I run this as may default user or admin role it will not work?